src/github.js

```diff
@@ -315,6 +315,12 @@
          this.createRepo = function (options, cb) {
             _request('POST', '/user/repos', options, cb);
          };
+
+         // Create an Organization repo
```

Comment:
Overall GitHub has a different set of API for organizations, so we should create this method differently. You can work on top of [this PR](https://github.com/michael/github/pull/289/files) that is already doing this change.